src/js/modules/infragistics.ui.editors.js

```diff
@@ -2777,7 +2777,7 @@
 				},
 				"compositionend.editor": function () {
 					setTimeout(function () {
-						var value, pastedValue, widgetName = self.widgetName,
+						var pastedValue, widgetName = self.widgetName,

```

Comment:
it's better to rename the variable from pastedValue - to compositionValue or similar.